README.md

```diff
@@ -327,6 +327,7 @@ Contributions are welcome. Please take a quick look at the [contribution guideli
  * [circuit_breaker](https://github.com/TPei/circuit_breaker) - Implementation of the circuit breaker pattern
  * [CrSignals](https://github.com/firejox/CrSignals) - Signals/slots notification library
  * [crystal-binary_parser](https://github.com/DanSnow/crystal-binary_parser) - Binary parser
+ * [crystal-i18n](https://github.com/crystal-i18n/i18n) - An internationalization library
```

Comment:
there is another one below i18n.cr. Could you please create a new section and move both?

Reply:
Sure thing! Just did it.